src/merklehash_p.js

```diff
@@ -241,9 +241,9 @@ class MerkleHash {
     }
 
     async readFromFile(fileName) {
-        const fd =await fs.promises.open(fileName, "r");
+        const fd = await fs.promises.open(fileName, "r");
         const header = new BigUint64Array(2);
-        await fd.read(header, {offset:0, length: 16, position:0});
+        await fd.read({buffer: header, offset:0, length: 16, position:0});
```

Comment:
does it still work when we upgraded to node 18.7 or later?

Reply:
It was added in: v13.11.0, v12.17.0 (https://nodejs.org/api/fs.html#filehandlereadoptions), and it is still in use, it works with version 18. We made same change on pilcom project one month ago. https://github.com/0xPolygonHermez/pilcom/commit/71d60918cb99a5020ef2bd014199062849e2dc75